### PR TITLE
Add man pages

### PR DESCRIPTION
**Pull request by AdminXVII**
Add man and html targets to the makefile, responsible for generating two versions of the content of the manpages directory, one for man per se and one for the web.
The man pages are a basic introduction to the window manager.
They are fairly short, as only one flag is supported and the specifications are already done by Awesome.
They are translated in french and english.

Issue: https://github.com/way-cooler/way-cooler/issues/335

### CODE REVIEW

**Comment by AdminXVII** [comment]
On the same track, it could be interesting to make a script to convert the website's documentation into a Texinfo manual. I can do it if you see it fit.

**Comment by Timidger** [comment]
Thanks for doing this work! I have a few nits about the actual content, but otherwise looks great!

> On the same track, it could be interesting to make a script to convert the website's documentation into a Texinfo manual. I can do it if you see it fit.

I'm not against that, though note that the current documentation on way-cooler.org is about the pre-awesome-clone API. Instead we will eventually have a clone of [the awesome docs](https://awesomewm.org/doc/).

File: manpages/awesome.1.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+awesome(1)
+==========
+
+NAME
+----
+
+awesome - Lua controller for Way-Cooler
```
Comment (Timidger) [inline manpages/awesome.1.md]: This wording is a bit ambiguous. Here's a clearer definition about what `awesome` does.

```suggestion
awesome - a clone of AwesomeWM as a Wayland client.
```

File: manpages/awesome.1.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+SYNOPSIS
+--------
+
+*awesome* [*--version*]
+
+DESCRIPTION
+-----------
+
+*way-cooler* is a tiling window manager for WayLand based on AwesomeWM.
```
Comment (Timidger) [inline manpages/awesome.1.md]: It's Wayland not WayLand

```suggestion
*way-cooler* is a tiling window manager for Wayland based on AwesomeWM.
```

**Comment by AdminXVII** [comment]
Done
If you have other requests for change in the future, I will be happy to change both versions

**Pull request merged**